jest.config.js

```diff
@@ -1,4 +1,4 @@
-const { pathsToModuleNameMapper } = require('ts-jest/utils');
+const { pathsToModuleNameMapper } = require('ts-jest');
```

Comment:
https://kulshekhar.github.io/ts-jest/docs/getting-started/paths-mapping/ newest way to importpathsToModulepathsToModule